app/views/external_users/admin/providers/edit.html.haml

```diff
@@ -2,26 +2,32 @@
 
 = render partial: 'layouts/header', locals: {page_heading: 'Edit provider details'}
 
-= form_for [:external_users, :admin, @provider], html: { novalidate: true } do |f|
+.body-content
+  = form_for [:external_users, :admin, @provider], html: { novalidate: true } do |f|
+    %fieldset
+      %legend
+        = 'Provider details'
+      .form-group
+        = f.label :name, t('.provider_name'), class: "form-label"
+        = validation_error_message(@provider, :name)
+        = f.text_field :name, class: "form-control"
 
-  %fieldset.user-form
-    = f.label :name, t('.provider_name'), class: "form-label"
-    = f.text_field :name
-    = validation_error_message(@provider, :name)
+      - if @provider.firm?
+        .form-group
+          = f.label :supplier_number, t('.supplier_number'), class: "form-label"
+          = f.text_field :supplier_number, class: "form-control"
+          = validation_error_message(@provider, :supplier_number)
 
-    - if @provider.firm?
-      = f.label :supplier_number, t('.supplier_number'), class: "form-label"
-      = f.text_field :supplier_number
-      = validation_error_message(@provider, :supplier_number)
+      - if @provider.firm?
+        .form-group
+          = f.label :vat_registered, class: "form-label"
```

Comment:
this would have to happen throughout the application. will need to be on another branch,